ndarray/src/main/java/org/tensorflow/ndarray/NdArray.java

```diff
@@ -285,33 +285,37 @@ default Stream<T> streamOfObjects() {
   NdArray<T> copyTo(NdArray<T> dst);
 
   /**
-   * Read the content of this N-dimensional array into the destination buffer.
+   * Copy the content of this N-dimensional array into the destination buffer.
    *
    * <p>The size of the buffer must be equal or greater to the {@link #size()} of this
    * array, or an exception is thrown. After the copy, content of the buffer and of the array can be
    * altered independently, without affecting each other.
    *
+   * <p><i>Note: this method was previously named `read(DataBuffer<T>)`, which led to confusion.</i>
```

Comment:
Maybe `in v0.4.0 and earlier this method...`, as just saying "previously" will look odd in a few years time when we can't remember when that was true.